src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -46,6 +46,12 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
     for (size_t hindex = 0; hindex < UARTDEV_COUNT_MAX; hindex++) {
```

Comment:
All uartHardware entries (UARTDEV_COUNT_MAX) in this PR are valid (USE_USARTx is not guarding them), so `uartDevice` (UARTDEV_COUNT) will  overflow if user configures more ports than defined by USE_UARTx